### PR TITLE
Feat/passport

### PR DESCRIPTION
**Pull request by acenine**

### CODE REVIEW

File: client/src/components/SignIn.jsx [hunk 1]
```diff
@@ -67,11 +83,18 @@ class SignIn extends React.Component {
 
       } else {
         // Failed Login 
+        console.log(data)
```
Comment (jGlass314) [inline client/src/components/SignIn.jsx]: Can we either remove this or something about it being a failed login? Just something like console.log('Failed Login:', data). Probably shouldn't print out the password that didn't work.

File: schema.sql [hunk 2]
```diff
@@ -4,9 +4,8 @@ CREATE DATABASE therebasebook;
 \c therebasebook;
 
 -- SERIAL- adds not null constraint, should increment by 1 for each new entry
-DROP TABLE IF EXISTS users CASCADE;
```
Comment (jGlass314) [inline schema.sql]: This DROP TABLE needs to stay in

File: client/src/components/SignIn.jsx (hunk 1 above)
Comment (acenine) [reply client/src/components/SignIn.jsx]: Yep I accidentally left that behind. It has been removed.

File: schema.sql [hunk 3]
```diff
@@ -4,9 +4,8 @@ CREATE DATABASE therebasebook;
...
 CREATE TABLE users (
-    id SERIAL PRIMARY KEY NOT NULL,
+    id SERIAL PRIMARY KEY NOT NULL UNIQUE,
```
Comment (jGlass314) [inline schema.sql]: we don't need UNIQUE. It's redundant. Primary Keys are always unique.
https://www.w3resource.com/PostgreSQL/primary-key-constraint.php

File: schema.sql [hunk 4]
```diff
@@ -19,9 +18,8 @@ INSERT INTO users (username, first_name, last_name, picture_url) VALUES ('rayang
 INSERT INTO users (username, first_name, last_name, picture_url) VALUES ('kmenghini', 'Kaitlyn', 'Menghini', 'https://lh4.ggpht.com/4nDELzdauqt2pyNaf-JI-ZDo6Ur87KgtQi9ASUaQF-l8qMIfufBXz0FLh1BV5oxGbDw=h900');
 INSERT INTO users (username, first_name, last_name, picture_url) VALUES ('sjain', 'Shubhra', 'Jain', 'https://petcube.com/blog/content/images/2017/04/kitten-supplies-cover-1.jpg');
 
-DROP TABLE IF EXISTS posts CASCADE;
```
Comment (jGlass314) [inline schema.sql]: This DROP TABLE needs to stay in

File: schema.sql [hunk 5]
```diff
@@ -19,9 +18,8 @@ INSERT INTO users (username, first_name, last_name, picture_url) VALUES ('rayang
...
 CREATE TABLE posts (
-    id SERIAL PRIMARY KEY NOT NULL,
+    id SERIAL PRIMARY KEY NOT NULL UNIQUE,
```
Comment (jGlass314) [inline schema.sql]: as before, we should leave out the UNIQUE.

File: schema.sql [hunk 6]
```diff
@@ -33,7 +31,6 @@ INSERT INTO posts (user_id, post_text) VALUES (3, 'ryan ngo post');
 INSERT INTO posts (user_id, post_text) VALUES (4, 'kaitlyn menghini post');
 INSERT INTO posts (user_id, post_text) VALUES (5, 'shubhra jain post');
 
-DROP TABLE IF EXISTS user_friends CASCADE;
```
Comment (jGlass314) [inline schema.sql]: Must leave DROP TABLE in. We can't drop a database in heroku.

File: server/api.js [hunk 7]
```diff
@@ -37,7 +46,7 @@ const api = {
     },
 
     getLikers: function (req, res) {
-
+      console.log("query text", req.query)
```
Comment (jGlass314) [inline server/api.js]: can we remove console.log?

**Review by jGlass314 — APPROVED** [review]

**Pull request merged**